civis/io/_tables.py

```diff
@@ -727,3 +765,48 @@ def f(x):
 
         fut.add_done_callback(f)
     return fut
+
+
+def _redshift_to_py(civ_client, tableid):
+
+    SQL_PANDAS_MAP = {
```

Comment:
I suggest moving this dictionary and `DATE_TYPES` into module-level constants. They seem like something worth putting at the top of the file.